experimenter/experimenter/nimbus-ui/src/lib/constants.ts

```diff
@@ -191,7 +192,7 @@ export const IMAGE_UPLOAD_ACCEPT = ".gif,.jpg,.jpeg,.png";
 export const POLL_INTERVAL = 30000;
 
 export const QA_STATUS_WITH_EMOJI = {
-  GREEN: ["✅ QA: Green", "success"],
-  YELLOW: ["⚠️ QA: Yellow", "text-dark"],
-  RED: ["❌ QA: Red", "danger"],
+  [NimbusExperimentQAStatusEnum.GREEN]: ["✅ QA: Green", "success", "✅"],
+  [NimbusExperimentQAStatusEnum.YELLOW]: ["⚠️ QA: Yellow", "text-dark", "⚠️"],
+  [NimbusExperimentQAStatusEnum.RED]: ["❌ QA: Red", "danger", "❌"],
```

Comment:
not sure if I like this completely 🤔 I'm open to suggestions if anyone has them

Reply:
beautiful! `Record<NimbusExperimentQAStatusEnum, QAStatusProperties>` looks like exactly what I am wanting 🤩 ty!